modular_skyrat/modules/customization/game/objects/items/conversion_kits.dm

```diff
@@ -0,0 +1,21 @@
+/obj/item/device/custom_kit
+    name = "modification kit" 
+    desc = "A box of parts for modifying a certain object."
+    icon = 'modular_skyrat/master_files/icons/donator/obj/kits.dmi'
+    icon_state = "partskit"
+    var/from_obj = null //The base object to be converted
+    var/to_obj = null //The object to turn it into
```

Comment:
As far as I know, you don't need to make them equal to null, declaring a variable without giving it anything makes it null too, I think. Try it out and tell me if it works.